src/Interpreters/loadMetadata.cpp

```diff
@@ -221,13 +221,26 @@ void loadMetadata(ContextMutablePtr context, const String & default_database_nam
         databases.emplace(default_database_name, std::filesystem::path(path) / escapeForFileName(default_database_name));
     }
 
+    bool has_ordinary_databases = false;
     TablesLoader::Databases loaded_databases;
     for (const auto & [name, db_path] : databases)
     {
         loadDatabase(context, name, db_path, has_force_restore_data_flag);
-        loaded_databases.insert({name, DatabaseCatalog::instance().getDatabase(name)});
+
+        auto database = DatabaseCatalog::instance().getDatabase(name);
+        loaded_databases.insert({name, database});
+        
+
+        if (database->getEngineName() == "Ordinary")
+            has_ordinary_databases = true;
+        
+        loaded_databases.insert({name,database});
     }
 
+    if (has_ordinary_databases)
+        context->addWarningMessage("Server Has Databases with Ordinary Engine which is not optimal. \n " 
```

Comment:
It will be good to tell the name of the first database with the Ordinary engine.